README.md

```diff
@@ -115,6 +110,7 @@ This section describes the available configuration options for the builder. Plea
 * `password` (string) The password to use when interfacing with the UpCloud API.
 * `zone` (string) The zone in which the server and template should be created (e.g. `nl-ams1`).
 * `storage_uuid` (string) The UUID of the storage you want to use as a template when creating the server.
+* `storage_name` (string) The name of the storage that will be used to find the first matching storage in the list of existing templates.
```

Comment:
are we introducing any breaking changes with this? 

If we are, would be clearer to just rename storage_uuid and storage_name to `storage`, and handle determine which one is passed in the driver

Reply:
otherwise would be good to state here 

Either of the following:

Reply:
>If we are, would be clearer to just rename storage_uuid and storage_name to storage, and handle determine which one is passed in the driver

better name would probably be `image`

Reply:
@tohalla nope, there are no breaking changes. You can pass `storage_uuid` or `storage_name` that will be used as a base for the image. 
Instead of renaming parameters and composing them into one, maybe better would be to move `storage_name` to the optional parameters list and extend help text?

Reply:
sounds good to me